include/mega/filesystem.h

```diff
@@ -266,7 +293,10 @@ struct MEGA_API FileSystemAccess : public EventTrigger
     void local2name(string*) const;
 
     // convert local path to MEGA format (UTF-8) with unescaping
-    void name2local(string*) const;
+    void name2local(string*, std::string *dstPath = nullptr) const;
```

Comment:
Can we make the 2nd parameter be `const std:;string*` please?  Initially I thought we were changing it to optionally output the result to the 2nd string instead of editing in place, like path2local etc.  thanks!

Reply:
Fixed in a0c21e3